pkg/informer/supportResources_test.go

```diff
@@ -0,0 +1,135 @@
+package informer
+
+import (
+	// "fmt"
+
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	// "k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+//generate fakeclient to get configmap
+//
+func TestGetConfigMapByName(t *testing.T) {
+	//Test scenarios:
+	// Correct ConfigMap - clientset1
+	// No configmap. -
+	// Configmap without AllowResources or DenyResources keys.- clientset2
+	// Configmap with invalid yaml in AllowResources or DenyResources. clientset3
+	// Configmap with invalid Name - clientset
+	configmaps := []struct {
+		clientset kubernetes.Interface
+		n         string
```

Comment:
Could we call these fields "name" and "namespace"? Personally, I think it makes the code more readable to have full words instead of abbreviations. It's more to type, but easier to interpret. 

Just a style preference. I'd be remiss if I didn't mention it.